application/Dockerfile

```diff
@@ -1,14 +1,12 @@
 FROM openjdk:16.0
 
-ENV WORK_DIR /usr/telestion
+ENV APP_DIR /usr/telestion/telestion-project-daedalus2
 
-EXPOSE 9870
+ADD build/distributions/telestion-project-daedalus2.tar /usr/telestion
 
-COPY conf/config.json $WORK_DIR/config.json
-COPY build/distributions/telestion-project-daedalus2.tar $WORK_DIR/
-COPY docker/exec.sh $WORK_DIR/
+WORKDIR $APP_DIR
+EXPOSE 9870
```

Comment:
Maybe it is better to expose only locally

Reply:
The `EXPOSE` option in the Dockerfile is more like a type of documentation that informs the user of the image that the image provide a service at this port.
Unfortunately, this option does not provide any bind option.
(cf. https://docs.docker.com/engine/reference/builder/#expose)

Reply:
Ah I see. Then it is okay. If you want to bind locally you can use the `ports` command.